README.rst

```diff
@@ -97,42 +121,148 @@ MVP2
+----------------------- ----------------- 
+2.8" `SPI display`_      N/A              
+----------------------- ----------------- 
+`micro:bit v2`_          2.x              
+----------------------- ----------------- 
+`joystick v2`_           2                
+======================= =================
+
+
+SW Configuration Index
+~~~~~~~~~
+
+======================= ================= ================================
+ Item                    Version            Comment
+======================= ================= ================================
+Windows 11 Enterprise    10.0.22631       N/A     
+----------------------- ----------------- --------------------------------
+Visual Studio Code       1.93.1           N/A
+----------------------- ----------------- --------------------------------
+nRF Connect for Desktop  v5.0.2           N/A
+----------------------- ----------------- --------------------------------
+Toolchain Manager        v1.5.2           N/A
+----------------------- ----------------- --------------------------------
+nRF Connect SDK          v2.6.2           N/A
+----------------------- ----------------- --------------------------------
+nRF Connect for VS Code  v2024.9.87       Can be downloaded using the Toolchain Manager found in nRF Connect for Desktop. Alternatively, it can be downloaded directly from inside Visual Studio Code.
+======================= ================= ================================
+
 Prerequisites
-"""""""""""""""""""""
+~~~~~~~~~
+
+#. Install the `Visual Studio Code`_.
+#. Install the `nRF Connect for VS Code`_.
 
 Build
-"""""""""""""""""""""
+~~~~~~~~~
+Game
+^^^^^^^^^
+* VS Code -> nRF Connect extension -> Add Folder as Application -> select zephyrdoom folder.
+* VS Code -> nRF Connect extension -> APPLICATIONS -> Add build configuration -> select board target nrf5340dk_nrf5340_cpuapp -> Build Configuration.
+Gamepad
+^^^^^^^^^
+* VS Code -> nRF Connect extension -> Add Folder as Application -> select gamepad\microbit folder.
+* VS Code -> nRF Connect extension -> APPLICATIONS -> Add build configuration -> select board target bbc_microbit_v2 -> Build Configuration.
 
 Flash
-"""""""""""""""""""""
+~~~~~~~~~
+Game
+^^^^^^^^^
+#. Connect Nordic Semiconductor `nRF5340`_ dev kit.
+#. Flash data (contains WAD file) to external flash::
+   
+     nrfjprog --family nrf53 --qspicustominit --program qspi.hex --verify
+#. Select game app.  VS Code -> nRF Connect extension -> APPLICATIONS -> Select zephyrdoom.
+#. Flash the game. VS Code -> nRF Connect extension -> ACTIONS -> Flash.
+
+Gamepad
+^^^^^^^^^
+#. Connect `micro:bit v2`_.
+#. Select gamepad app.  VS Code -> nRF Connect extension -> APPLICATIONS -> Select microbit.
+#. Copy file gamepad/microbit/build/zephyr/zephyr.hex to micro:bit (acting as a removable usb device).
 
 Monitor
-"""""""""""""""""""""
+~~~~~~~~~
+* VS Code -> nRF Connect extension -> CONNECTED DEVICES -> VCOM? COM ? -> Connect to Serial Port.
```

Comment:
It's always VCOM1 (VCOM0 is dedicated to the network core). The COM number is system dependent, so I would just put "VCOM1" and that's it.

Reply:
Updated